VRDR.Tests/Messaging_Should.cs

```diff
@@ -26,29 +26,48 @@ public void CreateSubmission()
         {
             DeathRecordSubmission submission = new DeathRecordSubmission();
             Assert.Equal("http://nchs.cdc.gov/vrdr_submission", submission.MessageType);
+            Assert.Null(submission.DeathRecord);
+            Assert.Equal("http://nchs.cdc.gov/vrdr_submission", submission.MessageDestination);
+            Assert.NotNull(submission.MessageTimestamp);
+            Assert.Equal("nightingale", submission.MessageSource);
```

Comment:
I'll confess I didn't notice this before... I'm not sure that nightingale is a good default for the source endpoint (or that there should be a default for the jurisdiction side of these messages).

Reply:
OK, I can remove this.